doc/rbd/rbd-mirroring.rst

```diff
@@ -13,14 +13,14 @@ capability is available in two modes:
   actual image. The remote cluster will read from this associated journal and
   replay the updates to its local copy of the image. Since each write to the
   RBD image will result in two writes to the Ceph cluster, expect write
-  latencies to nearly double when using the RBD journaling image feature.
+  latencies to nearly double while using the RBD journaling image feature.
```

Comment:
I agree with "while" in this context, and I also think that this sentence makes a pretty strong case for permitting split infinitives. I've always thought that, once we know the tradition of not splitting infinitives in the English language, we, by being willing to boldly split infinitives, open ourselves up to a world of new, extremely specific verbs that we previously would not have dared to... think.

Reply:
I see what you did there!

Sometimes stilted wording in the name of propriety is something up with which we should not put.